app/app355.cpp

```diff
@@ -88,31 +99,115 @@ int main(int argc, char* argv[])
         return 1;
     }
 
-    std::vector<TLE> tleVector;
-    std::string line;
-    int lineCount = 0;
-    while (std::getline(fileStream, line)) 
-    {
-        TLE newTLE{};
-        lineCount++;
+    std::list<OrbitalData> orbitalList;
+    std::string readLine;
 
-        if ((lineCount % 3) == 1)
+    std::string name = "";
+    std::string line1 = "";
+    std::string line2 = "";
```

Comment:
subjective: prefer to default construct `std::string`, rather than assign empty string literal